app/views/student_mailer/intent_reminder.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p>Hi <%= @user.name %>,</p>
+    <p>This is a reminder that you should submit your housing intent by <%= @intent_date %></p>
```

Comment:
Missing a period. I think it can also be combined with the next line into one `<p>` tag. Same with the locking reminder e-mail.